exchanges/binance/binance_websocket.go

```diff
@@ -400,8 +382,11 @@ func (b *Binance) wsHandleData(respRaw []byte) error {
 							err)
 					}
 
-					err = b.UpdateLocalBuffer(&depth)
+					init, err := b.UpdateLocalBuffer(&depth)
 					if err != nil {
+						if init {
```

Comment:
There is no scenario where `init` can return `true` and `err != nil`. 
Perhaps you meant to store the error from `err = b.obm.stageWsUpdate(wsdp, currencyPair, asset.Spot)` and use that? Because reading the code for `b.obm.checkIsInitialSync(currencyPair)`, the only scenario where `init` returns `true`, `err` is `nil`